### DISCUSSION TITLE
Select single but get data from cloud function

### DISCUSSION THREAD

**Discussion by mulyoved** [AntlerVC/firetable#720] [Support / Q&A] [answered]
Hi

Connector, even when "Allow for multiple item selection" is off, the result is an array of 1 item, I want to have only the item as the field value

Possible?

![image](https://user-images.githubusercontent.com/2309907/179920066-264de3dd-d549-4e71-8dab-9b4e3c63d358.png)

**Comment by BeeBombshell** [comment, marked as answer]
Hi @mulyoved, The connector field is a dropdown field that lets you pick values from another table, API, or a static list. Therefore, it has a datatype of an array. I am assuming you wish to have only a single item as the value of a particular field. For that, you could change the column type to any of the supported field types and set a default value for every entry.

For reference, you can check out our docs:
https://docs.rowy.io/field-types/supported-fields
https://docs.rowy.io/field-types/connector

If this doesn't help, it would be appreciated if you could provide a little more information about your usecase (perhaps a screen recording).
